_maps/map_files/generic/Lavaland.dmm

```diff
@@ -2881,7 +2881,7 @@
 	id_tag = "s_docking_airlock";
 	req_access_txt = "48"
 	},
-/obj/docking_port/mobile{
+/obj/docking_port/mobile/lockdown{
```

Comment:
I know you just changed this, but could you instead do a `/mining` subtype, then you could absorb all the var edits below into the subtype?

Reply:
Do you want a mining and labour camp subtype?

Reply:
yes

Reply:
Done!